main.js

```diff
@@ -104,22 +107,42 @@ async function main() {
     const $timeoutInput = $('#timeout-input');
     $timeoutRange.val(detectorParams.timeout);
     $timeoutInput.val(detectorParams.timeout);
+    if (Cookies.get('timeoutInputSet') === 'true' && Cookies.get('timeoutInputValue')){
+      let $timeoutValue = Number(Cookies.get('timeoutInputValue'));
+      $timeoutInput.val($timeoutValue);
+      $timeoutRange.val($timeoutValue);
+      detector.update({ timeout: $timeoutValue });
+    }
     $timeoutRange.change(event => {
       const value = event.target.value;
       $timeoutInput.val(value);
       detector.update({ timeout: value });
+      Cookies.set('timeoutInputSet', 'true', { expires: COOKIE_EXPIRATION_DAYS });
+      Cookies.set('timeoutInputValue', String(value), { expires: COOKIE_EXPIRATION_DAYS });
     });
     $timeoutInput.change(event => {
       const value = event.target.value;
       $timeoutRange.val(value);
       detector.update({ timeout: value });
     });
     const $heatmapInput = $('#heatmap-input');
+    if (Cookies.get('heatmap') === 'true'){
+      $heatmapInput.prop('checked', 'true');
+      detector.update({ renderPointCloud: true, renderHeatmap: true });
+    }
     $heatmapInput.change(event => {
       const value = $(event.target).is(':checked');
+      Cookies.set('heatmap', value, { expires: COOKIE_EXPIRATION_DAYS });
       detector.update({ renderPointCloud: value, renderHeatmap: value });
     });
     const $soundInput = $('#sound-input');
+    if (Cookies.get('alertAudio') === 'pop') {
+      $soundInput.val('pop');
+      alertAudio = new Howl({ src: [popUrl], html5: true });
+    } else if (Cookies.get('alertAudio') === 'coronavirus') {
+      $soundInput.val('coronavirus');
+      alertAudio = new Howl({ src: [coronavirusUrl], html5: true });
+    }
```

Comment:
Probably should move this to L41 when we initialize alertAudio. Also it would be nice if there a way to share the logic between here and the onChange because they look so similar.